convert2rhel/pkghandler.py

```diff
@@ -69,15 +69,28 @@ def call_yum_cmd_w_downgrades(cmd, fingerprints):
     return
 
 
-def call_yum_cmd(command, args="", print_output=True):
-    """Call yum command and optionally print its output."""
+def call_yum_cmd(command, args="", print_output=True, enable_repos=None, disable_repos=None):
```

Comment:
We need to allow overriding the `--releasever` in the `call_yum_cmd()` as the releasever we currently pass to yum is right for RHEL but does not apply to CentOS when installing subscription-manager.

```
[02/02/2021 23:20:32] DEBUG - Calling command 'yum install -y --releasever=7Server /usr/share/convert2rhel/subscription-manager/subscription-manager-1.24.42-1.el7.x86_64.rpm /usr/share/convert2rhel/subscription-manager/subscription-manager-rhsm-1.24.42-1.el7.x86_64.rpm /usr/share/convert2rhel/subscription-manager/subscription-manager-rhsm-certificates-1.24.42-1.el7.x86_64.rpm'
Loaded plugins: fastestmirror
Examining /usr/share/convert2rhel/subscription-manager/subscription-manager-1.24.42-1.el7.x86_64.rpm: subscription-manager-1.24.42-1.el7.x86_64
Marking /usr/share/convert2rhel/subscription-manager/subscription-manager-1.24.42-1.el7.x86_64.rpm to be installed
Examining /usr/share/convert2rhel/subscription-manager/subscription-manager-rhsm-1.24.42-1.el7.x86_64.rpm: subscription-manager-rhsm-1.24.42-1.el7.x86_64
Marking /usr/share/convert2rhel/subscription-manager/subscription-manager-rhsm-1.24.42-1.el7.x86_64.rpm to be installed
Examining /usr/share/convert2rhel/subscription-manager/subscription-manager-rhsm-certificates-1.24.42-1.el7.x86_64.rpm: subscription-manager-rhsm-certificates-1.24.42-1.el7.x86_64
Marking /usr/share/convert2rhel/subscription-manager/subscription-manager-rhsm-certificates-1.24.42-1.el7.x86_64.rpm to be installed
Resolving Dependencies
--> Running transaction check
---> Package subscription-manager.x86_64 0:1.24.42-1.el7 will be installed
--> Processing Dependency: python-dmidecode >= 3.12.2-2 for package: subscription-manager-1.24.42-1.el7.x86_64
Determining fastest mirrors


 One of the configured repositories failed (Unknown),
 and yum doesn't have enough cached data to continue. At this point the only
 safe thing yum can do is fail. There are a few ways to work "fix" this:

     1. Contact the upstream for the repository and get them to fix the problem.

     2. Reconfigure the baseurl/etc. for the repository, to point to a working
        upstream. This is most often useful if you are using a newer
        distribution release than is supported by the repository (and the
        packages for the previous distribution release still work).

     3. Run the command with the repository temporarily disabled
            yum --disablerepo=<repoid> ...

     4. Disable the repository permanently, so yum won't use it by default. Yum
        will then just ignore the repository until you permanently enable it
        again or use --enablerepo for temporary usage:

            yum-config-manager --disable <repoid>
        or
            subscription-manager repos --disable=<repoid>

     5. Configure the failing repository to be skipped, if it is unavailable.
        Note that yum will try to contact the repo. when it runs most commands,
        so will have to try and fail each time (and thus. yum will be be much
        slower). If it is a very temporary problem though, this is often a nice
        compromise:

            yum-config-manager --save --setopt=<repoid>.skip_if_unavailable=true

Cannot find a valid baseurl for repo: base/7Server/x86_64
CRITICAL - Failed to install subscription-manager packages. See the above yum output for details.
```